cli/cli/kurtosis_config/overrides_objects/v3/kurtosis_config_v3.go

```diff
@@ -0,0 +1,26 @@
+package v3
+
+import "github.com/kurtosis-tech/kurtosis/cli/cli/kurtosis_config/config_version"
+
+/*
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! WARNING !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+                           DO NOT CHANGE THIS FILE!
+  If you change this file, it will break config for users who have instantiated an
+           overrides file with this version of config overrides!
+    Instead, to make changes, you will need to add a new version of the config
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! WARNING !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+*/
+
+// NOTE: All new YAML property names here should be kebab-case because
+//  1. it's easier to read
+//  2. it's easier to write
+//  3. it's consistent with previous properties and changing the format of an already-written config file is very difficult
+type KurtosisConfigV3 struct {
+	// vvvvvvvvv Every new Kurtosis config version must have this key vvvvvvvv
+	ConfigVersion config_version.ConfigVersion `yaml:"config-version"`
+	// ^^^^^^^^^ Every new Kurtosis config version must have this key ^^^^^^^^
+
+	ShouldSendMetrics *bool                               `yaml:"should-send-metrics,omitempty"`
+	KurtosisClusters  map[string]*KurtosisClusterConfigV3 `yaml:"kurtosis-clusters,omitempty"`
+	CloudConfig       *KurtosisCloudConfigV3              `yaml:"cloud-config,omitempty"`
```

Comment:
Sooo, if that's a pointer with `omitempty` do you need all this migration jazz?? I guess a nullable pointer is backward compatible in this case, right?